server/testutils/factories/cas1SpaceBookingSummary.ts

```diff
@@ -0,0 +1,15 @@
+import { Factory } from 'fishery'
+import { faker } from '@faker-js/faker'
+import { Cas1KeyWorkerAllocation, Cas1SpaceBookingSummary, PersonSummary } from '@approved-premises/api'
+import { fullPersonSummaryFactory } from './person'
+import staffMemberFactory from './staffMember'
+import { DateFormats } from '../../utils/dateUtils'
+
+export default Factory.define<Cas1SpaceBookingSummary>(() => ({
+  id: faker.string.uuid(),
+  person: fullPersonSummaryFactory.build() as PersonSummary,
+  canonicalArrivalDate: DateFormats.dateObjToIsoDate(faker.date.soon({ days: 90 })),
+  canonicalDepartureDate: DateFormats.dateObjToIsoDate(faker.date.soon({ days: 365 })),
```

Comment:
If the departure date needs to absolutely be _after_ the arrival date (though I'm not sure the tests care that much beyond displaying it), you can use the `refDate` option in the `faker.date.soon` call. Something along those lines:

```js
const arrival = faker.date.soon({ days: 90 })
const departure = faker.date.soon({ days: 365, refDate: arrival })
```